src/etc/schema.graphqls

```diff
@@ -104,13 +105,10 @@ input SaveAddressInformation {
 }
 
 type QuoteData implements TotalsObject  {
-    id: Int
-    items_count: Int
-    customer_is_guest: Boolean
-    store_id: Int
-    customer_tax_class_id: Int
-    items_qty: Float
-    coupon_code: String
+    is_virtual: Boolean
+}
```

Comment:
Is this 100% necessary? It might introduce breaking changes.

Reply:
This will not, this just exposes properties of the same totals object. Remember the issue when we wanted to use the totals returned from checkout in cart? This allows for full comparability.